app/src/main/res/layout/fragment_helper_request_overview.xml

```diff
@@ -41,14 +41,21 @@
         android:layout_height="0dp"
         android:layout_marginBottom="10dp"
         android:layout_weight="1"
-        tools:listitem="@layout/buyer_request_row" />
+        tools:listitem="@layout/row_user" />
 
-    <TextView
+    <Button
+        android:id="@+id/button_nearRequests"
+        style="@style/Widget.AppCompat.ActionButton"
+        android:drawableEnd="@drawable/ic_edit_black_24dp"
+        android:drawableTint="@android:color/white"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="@string/helper_request_overview_heading_available_section"
+        android:paddingStart="0dp"
+        android:paddingEnd="0dp"
+        android:textAlignment="textStart"
+        tools:text="Available requests at 71323"
         android:textColor="@android:color/white"
-        android:textSize="25sp"
+        android:textSize="@dimen/textSize_screen_title"
```

Comment:
even if dimens are kinda cool feature, they need a lot of maintenance in bigger projects. Defining dimention values directly in the layout is easier to change and maintain. we should use them only if they change based on screen size etc.